tests/integration/security/rbac/v2/basic/extended_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Istio Authors
```

Comment:
I think we need better naming than `extended_test.go`, it's very easy to get confused what is the `basic_test` and what is `extended_text`, and not clear what's the difference for the test cases in them.

Also it might be a better idea to put this in a separate directory so that we group the tests by the features it's testing, like we have a directory for `group`.

Reply:
Do you have any suggestion for the name?

Reply:
As for the separate directory, since these tests are testing new features of RBAC v2, I think we should put them in the same directory to be in the same test suite (https://github.com/istio/istio/tree/master/tests/integration#adding-new-tests). Otherwise, we will have a lot of duplicated setup code, and we have to deploy Istio for each test, which is pretty consuming.

Reply:
Could you guys share some input on this? @ozevren @nmittler

Reply:
We should try to co-locate tests in the same folder as much as we can. This will ensure that a single suite-level setup can be used for many tests and reduce runtime of tests.

Reply:
Okay, then let's keep it in the same directory.